lib/api/objectPutCopyPart.js

```diff
@@ -259,9 +259,9 @@ function objectPutCopyPart(authInfo, request, sourceBucket,
                                         { error: err });
                                         return cb(errors.InternalError);
                                     }
-                                    return data.put(cipherBundle, hashedStream,
-                                        numberPartSize, dataStoreContext,
-                                        backendInfo, log,
+                                    return data.put(cipherBundle,
+                                        hashedStream, numberPartSize,
+                                        dataStoreContext, backendInfo, log,
```

Comment:
these just look like line break changes in this file so please remove.  separate pr for copy part?